### ISSUE TITLE
Launching from Google Chrome causes mpv to crash

### ISSUE THREAD

**Issue by aleasto** [UnitedRPMs/acestream-launcher#3]
`mpv: symbol lookup error: mpv: undefined symbol: vkCreateWaylandSurfaceKHR`

Running xdg-open directly from terminal works fine. Launching from firefox works fine. What is chrome doing?

**Comment by sergiomb2** [comment]
in what version of Fedora ? is this problem https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/HSRJZ4TMBW2F5U2MGOMVWCFBVJ2NURVO/ ?

**Comment by sergiomb2** [comment]
> What is chrome doing?

I also had some issues with Chrome ...

**Comment by aleasto** [comment]
Fedora 35.

Though some packages were never rebuilt for fc35:
```
acestream-engine-3.1.49-6.fc34.x86_64
openjpeg-libs-1.5.1-29.fc34.x86_64
python2-setuptools-41.2.0-4.fc34.noarch
```
I don't know if this used to work in previous fedora versions.

The error also happens in an x11 environment, so i'll update the title.

**Comment by aleasto** [comment]
Sorry, this looks to be caused by rpmfusion build of mpv.
Switching to unitedrpm build downgraded some dependencies but fixed the crash.

**Comment by aleasto** [comment]
`libvulkan.so.1 => /opt/google/chrome/libvulkan.so.1 (0x00007f65435b2000)`

Launching from chrome is making it use their libvulkan. No clue why it works with unitedrpm mpv.
In both cases `readelf -s` shows:
```
Num:    Value          Size Type    Bind   Vis      Ndx Name
550: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND vkCreateWaylandSurfaceKHR
```

**Comment by 2-4601** [comment]
I've never used this program but found out this issue through a Google search.

In my case I'm using https://github.com/SanderRonde/CustomRightClickMenu to open `mpv` via Google Chrome and I'm getting the same error:
```
mpv: symbol lookup error: mpv: undefined symbol: vkCreateWaylandSurfaceKHR
```

I'm using a Google Chrome package from the AUR which is basically the Debian package that Google maintains (with a few Arch Linux specific patches). The .deb comes with Google's own version of `libvulkan.so.1` which does not have `vkCreateWaylandSurfaceKHR` (as described in the comment above). I was able to solve this by linking the Google supplied `libvulkan.so.1` to the Arch supplied one (provided by [`vulkan-icd-loader`](https://archlinux.org/packages/extra/x86_64/vulkan-icd-loader/)). This solved the issue for me.

Edit: See https://github.com/UnitedRPMs/acestream-launcher/issues/3#issuecomment-993775576 for a better solution.

**Comment by sergiomb2** [comment]
ah or just remove /opt/google/chrome-beta/libvulkan.so.1 , worked for me .

Anyway I don't use wayland

**Comment by aleasto** [comment]
Yeah i have proposed this supid _fix_ upstream: https://github.com/jonian/acestream-launcher/pull/44
I don't know if you're interested in picking it up for fedora.

**Comment by 2-4601** [comment]
I took a look at your PR and it inspired me to create a more maintainable solution for my use case:
I created a wrapper shell script for `mpv`, which simply unsets `LD_LIBRARY_PATH` and then launches `mpv`.

**Comment by miXwui** [comment]
I don't use acestream or whatever this repo is for, but have the same issue trying to launch a video in mpv from Gnome Nautilus on Sway, and it seems to be a Wayland/Vulkan issue? I got here from a web search so leaving this in case it helps someone else.

I got inspiration from this thread, thanks!

Error in journal:

```text
 mpv: symbol lookup error: mpv: undefined symbol: vkCreateWaylandSurfaceKHR
```

Setting/clearing `LD_LIBRARY_PATH=` in `/usr/share/applications/mpv.desktop` fixed the issue for me:

```
...
Exec=env LD_LIBRARY_PATH= mpv --player-operation-mode=pseudo-gui -- %U
...
```
